DatabaseAndReporting/Integration/01_ManageQueries.md

```diff
@@ -0,0 +1,131 @@
+
+8. Select the *Public download enabled* checkbox to allow any user, also non-logged in users, with the applicable link to execute the query and download the retrieved data via web browser. 
+    > [Info] A specific download link, containing a UUID (Universally Unique Identifier), is required to execute the query in a web browser.
+
+9. Select the *Write access enabled* checkbox to allow write operations when executing the query. Otherwise, a query with write operations, such as INSERT or DELETE, will not be executed and an error message will be displayed.
+
+10. If desired, click the [TRY QUERY (READ ONLY)] button to test the query.  
+    The query results are displayed in table format at the bottom of the workspace.
+
+11. Click the [SAVE] button.  
+   The query has been saved. The *Create MySQL query* is closed. The new query is displayed in the list of queries.
+
+
+## Edit a query
+
+After you have a created a query, it can be edited to change any previously set values. The fields *ID* and *UUID*, however, are automatically assigned by the system and cannot be modified.
+
+#### Prerequisites 
+
+A query has been created, see [Create a query](#create-a-query).
+
+#### Procedure
+
+*Database and reporting > Managed queries > Tab QUERIES*
+
+![Queries](../../Assets/Screenshots/DatabaseAndReporting/ManagedQueries/Queries/Queries.png "[Queries]")
+
+
+1. Click the query to be edited in the list of queries. Alternatively, select the checkbox of the query to be edited and click the ![Edit](../../Assets/Icons/Edit01.png) (Edit) button in the editing toolbar.  
+    The *Edit MySQL query* view is displayed.
+
+    ![Queries](../../Assets/Screenshots/DatabaseAndReporting/ManagedQueries/Queries/EditQuery.png "[Queries]")
+
+2. Edit the query set values as necessary in the corresponding fields and checkboxes.
+    
+3. If desired, click the [TRY QUERY (READ ONLY)] button to test the query.  
+  The query results are displayed in table format at the bottom of the workspace.
+
+4. Click the [SAVE] button.  
+  The changes have been saved. The *Edit MySQL query* is closed. 
+
+
+## Delete a query
+
+Delete a query that is no longer needed. It is possible to delete several queries at once.  
+
+#### Prerequisites 
+
+At last a query has been created, see [Create a query](#create-a-query).
+
+#### Procedure
+
+*Database and reporting > Managed queries > Tab QUERIES*
+
+![Queries](../../Assets/Screenshots/DatabaseAndReporting/ManagedQueries/Queries/Queries.png "[Queries]")
+
+
+1. Select the checkbox of the query to be deleted.   
+    The editing toolbar is displayed.
+
+2. Click the ![Delete](../../Assets/Icons/Trash03.png) (Delete) button in the editing toolbar.  
+    The *Deleted queries xxxx* pop-up window is displayed. The number indicates the identifier of the deleted query as displayed in the *ID* column. The deleted query is removed from the list of queries. 
```

Comment:
welche number? xxxx = number? Dann eher .... The *xxxx* indicates the identifier...